### PR TITLE
updated readme.md with correct grammar

### PR DESCRIPTION
**Pull request by ritikagrawat**
made following changes:
nginx- Nginx
in to- Into
by passing- bypassing

### CODE REVIEW

**Comment by fredemmott** [comment]
Hi; sorry, I'm guessing this is trivial, but I'm not able to look at the content without the CLA.

**Comment by sophiawisdom** [comment]
"by passing" -> "bypassing" is wrong, "bypassing something to a program" doesn't make sense.

File: README.md [hunk 1]
```diff
@@ -21,10 +21,10 @@ You can install a [prebuilt package](https://docs.hhvm.com/hhvm/installation/int
 
 ## Running
 
-You can run standalone programs just by passing them to hhvm: `hhvm example.hack`.
+You can run standalone programs just bypassing them to hhvm: `hhvm example.hack`.
```
Comment (ecreeth) [inline README.md]: is "by passing"

**Comment by fredemmott** [comment]
Sorry, closing; unable to review without CLA.

**Pull request closed without merging**